backend/config/error_handlers.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+
+
+def handler500(request, template_name="500.html"):
+    """
+    Custom handler for 500s. Ensures the 500 template loads with proper context (environment, login info, OMB values).
+    """
+    return render(request, template_name, {})
```

Comment:
if we're passing an empty context dict here, where/how does the mentioned proper context come into play?

Reply:
Django's default 500 view explicitly renders the `500.html` template with no variables. By overriding it here (even if we don't give it anything in particular), we allow the template to render with the global context added by the functions in `context_processors.py`.

Reply:
makes sense- thanks @jperson1!